### PR TITLE
Change param name

### LINKED ISSUE #1: Add blog ID to the response object, and rename to `blog`
The query parameter `blog_id` is a bit too verbose; this should be renamed to `blog`. In addition, this should be added to the response object.

### PR DESCRIPTION
**Pull request by roborourke**
Change param name and return it in the data if sent

Fixes #1

### CODE REVIEW

File: class-rest-api.php [hunk 1]
```diff
@@ -199,7 +200,7 @@ public function delete_item( $request ) {
 	 */
 	protected function prepare_item_for_database( $request ) {
 
-		$update         = new \stdClass;
+		$update = new \stdClass;
```
Comment (joehoyle) [inline class-rest-api.php]: `new stdClass`, `use stdClass` FTW!

File: class-rest-api.php [hunk 2]
```diff
@@ -262,15 +263,23 @@ public function get_item_schema() {
...
 	 * Convert a Mapping object to data we can return
 	 *
-	 * @param Mapping $mapping
+	 * @param Mapping         $mapping
+	 * @param WP_REST_Request $request
 	 * @return array
 	 */
-	protected function mapping_to_data( $mapping ) {
-		return array(
+	protected function mapping_to_data( $mapping, $request ) {
```
Comment (joehoyle) [inline class-rest-api.php]: `prepare_item_for_response` is the typical name we have for this method, not required of course

File: class-rest-api.php [hunk 3]
```diff
@@ -280,7 +289,7 @@ protected function mapping_to_data( $mapping ) {
 	 * @return int
 	 */
 	protected function get_blog_id( $request ) {
-		return $request->get_param( 'blog_id' ) ?: get_current_blog_id();
+		return $request->get_param( 'blog' ) ?: get_current_blog_id();
 	}
 
 }
```
Comment (joehoyle) [inline class-rest-api.php]: Newline at end of file.

File: class-rest-api.php (hunk 2 above)
Comment (roborourke) [reply class-rest-api.php]: @joehoyle this method is called by `prepare_item_for_response` :)

Comment (joehoyle) [reply class-rest-api.php]: Oops :P sorry

Comment (roborourke) [reply class-rest-api.php]: Actually yeah I can't remember why I split them now...

**Comment by roborourke** [comment]
@joehoyle should be good now, updated the schema too

**Pull request merged**